packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -170,15 +169,13 @@ class IconButton extends StatelessWidget {
 
   /// The size of the icon inside the button.
   ///
-  /// This property must not be null. It defaults to 24.0.
```

Comment:
This still defaults to 24 if everything else is null. That should still be documented.

Reply:
fixed in the new commit